GetIntoTeachingApi/Models/Validators/GetIntoTeachingCallbackValidator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using FluentValidation;
+using GetIntoTeachingApi.Services;
+
+namespace GetIntoTeachingApi.Models.Validators
+{
+    public class GetIntoTeachingCallbackValidator : AbstractValidator<GetIntoTeachingCallback>
+    {
+        public GetIntoTeachingCallbackValidator(IStore store, IDateTimeProvider dateTime)
+        {
+            RuleFor(request => request.AcceptedPolicyId).NotNull();
+
+            RuleFor(request => request.Email).NotEmpty();
+            RuleFor(request => request.FirstName).NotEmpty();
+            RuleFor(request => request.LastName).NotEmpty();
+            RuleFor(request => request.AddressTelephone).NotNull();
+            RuleFor(request => request.PhoneCallScheduledAt)
+                .NotNull()
+                .GreaterThan(candidate => dateTime.UtcNow)
+                    .WithMessage("Can only be scheduled for future dates.");
```

Comment:
This was intentional to show that the message applies to the `GreaterThan` rather than the `NotNull`, although looking at it I'm not sure if it actually applies to _both_ or just the last one (I expected the last one but I'll check)

Reply:
Yeah it only applies to the `GreaterThan` ^